cmd/server/handler.go

```diff
@@ -60,9 +59,6 @@ func EnhanceMiddleware(ctx context.Context, sl *servicelocatorx.Options, d drive
 	}
 
 	n.UseHandler(router)
-	corsx.ContextualizedMiddleware(func(ctx context.Context) (opts cors.Options, enabled bool) {
-		return d.Config().CORS(ctx, iface)
-	})
```

Comment:
This function returns a middleware, but it was never used. So CORS is not working on master? Except for the per-client CORS?

Reply:
🤔